aws/lambda/python/sns_converter.py

```
@@ -5,36 +5,47 @@
-    
+
     message_to_sns = {
         "version": "1.0",
         "source": "custom",
         "content": {
-            "description": f"{warning_string} {event_source_string} {bucket_name_string} {file_name_string} {user_identity_id_string} {event_name_string} {buckets_link}",
+            "description": (
+                f"{warning_string} {event_source_string} {bucket_name_string} {file_name_string} "
+                f"{user_identity_id_string} {event_name_string} {buckets_link}"
+            ),
         }
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)